### ISSUE TITLE
You already have the latest version of settings

### ISSUE THREAD

**Issue by fredrikaverpil** [shanalikhan/code-settings-sync#221]
**Visual Studio Code Version :** Version 1.10.0-insider (1.10.0-insider)
**Code Settings Sync Version :** 2.4.3
**Operating System :** Windows 10 / macOS Sierra
**Occurs On:** Upload / Download
**Proxy Enabled:** No
**GIst Id:** b60e2f8cd23dbc1728e4dba712d72032

I've installed vscode on my Windows machine and on my macOS machine. They both use at the same gist + token.

If I make a change in the settings on my Windows machine and then upload those settings, I can see my settings are being synced properly and the gist is updated over at Github. All good so far.

If I then choose to download my settings on the mac, it'll just say that I already have the latest settings and won't download. However, this settings.json does not have the change which I performed on the Windows machine. In my mind, it should download the new settings which clearly is available on the gist.

Do you see this on your end too?

I notice that `sync.lastDownload` looks weird in my settings.json on both machines. Could that be related?

```json
    "sync.gist": "b60e2f8cd23dbc1728e4dba712d72032",
    "sync.lastUpload": "2017-02-17T21:03:51.813Z",
    "sync.autoDownload": false,
    "sync.autoUpload": false,
    "sync.lastDownload": "1970-01-01T00:00:00.000Z",
    "sync.version": 243,
    "sync.showSummary": true,
    "sync.forceDownload": false,
    "sync.workspaceSync": false,
    "sync.anonymousGist": false,

```

**Comment by xtongs** [comment]
Having the same problem when switching to a new Mac, I just set the 'sync.forceDownload' to true and then the settings can be downloaded on the new machine. Don't know why the problem happens, but if you need the settings right now you can try this. Then may be set forceDownload to false in case there will be other problems in the future.

**Comment by shanalikhan** [comment]
@fredrikaverpil i see you are using Insiders, i have just released a update v2.4.4 fixing this problem.
Let me know if there is any problem like this, if there is not i will release it to public in next few days.

**Comment by fredrikaverpil** [comment]
This now works flawlessly in 4.2.2 for me, without any issues.
